package-lock.json

```diff
@@ -24,7 +24,7 @@
                 "simple-git": "3.5.0",
                 "sql-formatter-plus": "1.3.6",
                 "toposort": "2.0.2",
-                "update-notifier-git": "5.0.3",
+                "update-notifier": "5.1.0",
```

Comment:
could you run a npm prune again to ensure we only have changes from this branch in here?